samples/web/app.rb

```diff
@@ -66,14 +66,12 @@ def resize(url, width)
 #
 post('/signin') do
   audience = settings.client_id.id
-  # Important: The google-id-token gem is not production ready. If using, consider fetching and
-  # supplying the valid keys separately rather than using the built-in certificate fetcher.
   validator = GoogleIDToken::Validator.new
-  claim = validator.check(params['id_token'], audience, audience)
+  claim = validator.check(params['credential'], audience, audience)
```

Comment:
We should probably also remove the dependency on `google-id-token` and instead use the classes in googleauth.